### ISSUE TITLE
Distortion measures

### ISSUE THREAD

**Issue by ccasam** [eth-igl/GP2020-Assignments#23]
Hi! :)
I am looking at point 3.2 of assignment 4. Could we recap the different distortion measures it makes sense to compute? I will divide the questions in three points so it is maybe easier to answer.

1) If J is the transformation that maps a triangle (2D in its local coordinate framework) to the 2D parameterization, we saw that the singular values of J (sigma and SIGMA) are related to length distortion. In particular we saw two distortion measures: sqrt(sigma^2 + SIGMA^2) and max(SIGMA, 1/sigma). These two measures are both related to length distortion, is that correct?

2) What if we want to measure angle distortion? Should we maybe look at the modulus of the off-diagonal entries of J^TJ (= first fundamental form)? Or is there a specific measure of angle distortion we should use? I did not find it in the slides.

3) Or maybe as a distortion measure should we show directly the quantity that we are trying to minimize in the different exercises (e.g. use ||J-R||^2 as a distortion measure since this is what we are minimizing in ARAP)? This third option seems actually a bit less interpretable to me.

To summarize: what distortion measures does it make sense to use? What is a good measure of angle distortion?

Thank you very much!

**Comment by PhHerholz** [comment]
1) J is only the linear part. There is also a translation, however, it is irrelevant for distortion measures. Regarding the distortion measures: consider the SVD 
![image](https://user-images.githubusercontent.com/26648705/79729631-01ab8000-82f0-11ea-94a1-30b2f1215662.png)
The matrices U and V are orthogonal so we have a decomposition into rotation - anisotropic scaling - rotation. From this we can see that for \Sigma = \sigma = 1 we have a pure rotation which is optimal in terms of distortion. Moreover, we can think of \Sigma and \sigma as the maximal and minimal stretch along a direction in the triangle, respectively.
So, yes, these measures are closely related to (length) distortion.
2) We commonly measure angle distortion indirectly by observing that if J is a rotation (= orthogonal matrix without reflection), possibly with istropic scaling, we have perfect angle preservation.  The distortion measure 
![image](https://user-images.githubusercontent.com/26648705/79730308-f3aa2f00-82f0-11ea-8b17-ad6350664673.png)
achieves exactly this.

3) The idea of the exercise is to experiment with the measures. You should be able to interpret the different measures and their color coding. It's not so much about right or wrong, just pick measures that show the different features of the parameterization techniques.
